README.md

```diff
@@ -104,6 +104,16 @@ algorithm:
 
 [go-env]: https://golang.org/cmd/go/#hdr-Environment_variables
 
+## Custom IPFS gateway URL
+
+By default, `ipfs-update` uses https://ipfs.io as the gateway URL. If you wish to use your own IPFS gateway URL, please export it via the environment variable `IPFS_CUSTOM_GATEWAY_URL`.
+
+For example:
+
+```sh
+$ IPFS_CUSTOM_GATEWAY_URL="https://dweb.link" ipfs-update install latest
```

Comment:
nit: we usually use `IPFS_GATEWAY` (at least that's what I have set in my local environment).

Reply:
I have updated the README. Thanks for throwing some light on this, didn't realize that a env. var. was already under use.